### PR TITLE
doc: fixed building command for VEGAboard

### PR DESCRIPTION
**Pull request by howardliu7874**
Signed-off-by: Howard Liu <howardliu7874@hotmail.com>

### CODE REVIEW

**Review by dbkinder — APPROVED** [review]
doc changes LGTM.  @mbolivar you agree?

File: boards/riscv32/rv32m1_vega/doc/index.rst [hunk 1]
```diff
@@ -573,7 +573,7 @@ Windows (run this in a ``cmd`` prompt, from the Zephyr directory)::
 
   # Use CMake to generate a Ninja-based build system:
   type NUL > empty.ld
-  cmake -GNinja -DBOARD=rv32m1_vega_ri5cy -DCMAKE_REQUIRED_FLAGS=-Wl,-dT=empty.ld ..
+  cmake -GNinja -DBOARD=rv32m1_vega_ri5cy -DCMAKE_REQUIRED_FLAGS=-Wl,-dT=../../empty.ld ..
```
Comment (mbolivar) [inline boards/riscv32/rv32m1_vega/doc/index.rst]: Hm, if it's a path issue, why not `%cd%\empty.ld` or so?

Comment (howardliu7874) [reply boards/riscv32/rv32m1_vega/doc/index.rst]: Good point. I'll use it. thanks.

**Review by mbolivar — APPROVED** [review]
Thanks for the fix! LGTM

**Review by MaureenHelm — APPROVED** [review]
Thanks @howardliu7874 !

**Pull request merged**